tests/common/helpers/constants.py

```diff
@@ -45,7 +45,8 @@
     "t2": "t1",
     "m0_vlan": "server",
     "m0_l3": "mx",
-    "ft2": "lt2"
+    "ft2": "lt2",
+    "lt2": "t1"
```

Comment:
You may also need to add support for LT2 in `UPSTREAM_ALL_NEIGHBOR_MAP`

Reply:
Thanks, I've updated FT2 as well